sys/include/crypto/modes/cbc.h

```diff
@@ -20,6 +20,7 @@
 #ifndef CRYPTO_MODES_CBC_H
 #define CRYPTO_MODES_CBC_H
 
+#include <stddef.h>
```

Comment:
Is this really needed ?

Reply:
I tought RIOT's header files should be stand alone. This header file only includes `ciphers.h` which only includes `stdint.h`.

More practical: I removed RIOT's `assert.h` in order to test with c11 `static_assert` and I got an compilation error in this header because `size_t` wasn't defined in my app.

Reply:
ok